scss/detail.scss

```diff
@@ -0,0 +1,362 @@
+@import 'variables';
+
+.body{
+  font-family: 'Roboto', sans-serif;
+}
+
+.master-container{
+  display: flex;
+  height: 768px;
+  width: 1366px;
+  background-color: $wild-sand;
+  justify-content: center;
+  align-items: center;
+
+  .return-button{
+    display: flex;
+    flex-direction: row;
+    color: $black;
+    font-size: 20px;
+    line-height: 24px;
+    position: absolute;
+    top: 46px;
+    right: 1042px;
+    bottom: 702px;
+    left: 265px;
+    content: "book-list.html";
+  }
+
+  .return-button::before{
+    margin-right: 12px;
+    content: url(https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f0a/x/e42f9110cec094df3e49702baa8263f7/back-arrow-20.png);
+  }
+
+  .navbar-container{
+    position: fixed;
+    top: 0;
+
+    .border{
+      height: 6px;
+      width: 1366px;
+      background-color: $cerulean;
+    }
+
+    .navbar{
+      height: 70px;
+      width: 1366px;
+      background-color: $white;
+      box-shadow: 0 4px 10px 0 rgba(0,0,0,0.5);
+
+      .wolox-logo{
+        position: absolute;
+        left: 246px;
+        right: 908px;
+        height: 70px;
+        width: 212.42px;
+      }
+
+      .logout{
+        position: absolute;
+        top: 26px;
+        right: 246px;
+        bottom: 26px;
+        left: 1057px;
+        color: $black;
+        font-size: 18px;
+        line-height: 21px;
+      }
+    }
+  }
+
+  .master-book-list-container{
+    display: flex;
+    height: 636px;
+    width: 900px;
+    background-color: $wild-sand;
+    flex-flow: column wrap;
+    justify-content: space-around;
+    align-items: center;
+    margin-top: 36px;
+  
+    .book-container{
+      display: flex;
+      height: 300px;
+      width: 198px;
+      background-color: $white;
+      box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+      flex-direction: column;
+      justify-content: center;
+  
+      .cover-page-list{
+        height: 200px;
+        width: 142px;
+        align-self: center;
+      }
+  
+      .book-list-detail{
+        display: flex;
+        flex-direction: column;
+        margin-top: 12px;
+        padding-left: 28px;
+        
+  
+        .book-list-title{
+          color: $black;
+          font-size: 16px;
+          font-weight: 700;
+          line-height: 18px;
+          margin-bottom: 8px;
+        }
+  
+        .book-list-autor{
+          color: $gray;
+          font-size: 14px;
+          line-height: 18px;
+        }
+      }
+    }
+
+    .book-container:hover{
```

Comment:
You can nest this hover selector inside `.book-container` using the parent selector (&)